src/bitcoin-cli.cpp

```diff
@@ -1028,6 +1080,64 @@ static void ParseGetInfoResult(UniValue& result)
         result_string += "\n";
     }
 
+    const double mempool_bytes{result["mempool_bytes"].get_real()};
+
+    if (mempool_bytes >0){
+
+    	result_string += strprintf("%sMempool:%s\n", MAGENTA, RESET);
+
+    	std::string fee_rate;
+    	std::string size_bar;
+    	double fee_size = 0;
+
+    	double size1 = result["feerate_dist1"].get_real();
+    	double size2 = result["feerate_dist2"].get_real();
+    	double size3 = result["feerate_dist3"].get_real();
+    	double size4 = result["feerate_dist4"].get_real();
+
+    	//Removing zero from numbers used in std::min to avoid division by zero in GetFeerateBar()
+    	auto const ignore_zero = [](auto const& a, auto const& b) -> bool {
+        	if(0 == a || 0 == b)
+        	{
+            	return false;
+        	}
+        	return a < b;
+    	};
+
+    	double max_size = std::max({size1, size2, size3, size4});
+    	double inc_size = std::min({size1, size2, size3, size4}, ignore_zero);
```

Comment:
What should happen when all values are zero?

Reply:
`mempool_bytes` will be zero in that case and mempool section will not be printed L1085